pkg/monitoragent/cell_linux.go

```diff
@@ -73,12 +72,6 @@ func newMonitorAgent(params agentParams) ciliumagent.Agent {
 						queueSize = defaults.MonitorQueueSizePerCPUMaximum
 					}
 				}
-
-				monitorErr := ciliumagent.ServeMonitorAPI(ctx, agent, queueSize)
```

Comment:
Why are we removing this?

Reply:
cilium stops sending events to the socket if this isnt removed
i believe it's overwriting the cilium connection so then we stop seeing events from actual cilium daemon